benchmarks/2019.json

```diff
@@ -5101,14 +5173,32 @@
     "isToppedOut": false,
     "deciles": [
       0,
-      21.19,
-      28.1,
-      34.78,
-      38.17,
-      41.89,
-      46.54,
-      58.88,
-      67.15
+      20.83,
+      24.56,
+      31.82,
+      38.1,
+      41.46,
+      44.62,
+      47.83,
+      65.25
+    ]
+  },
+  {
+    "measureId": "ACRAD23",
```

Comment:
Previous
```
{"level":"debug","message":"measure ACRAD23 2017 submission method registry single_performance_rate_measurements: 69 measurement-stratas","timestamp":"2019-01-03T23:36:56.976Z"}
{"level":"debug","message":"A total of 46 measurements may be in decile calculation for measure ACRAD23 registry 4","timestamp":"2019-01-03T23:36:56.976Z"}
{"level":"debug","message":"Success! measure ACRAD23 has 46 qualified TIN-NPIs reporting for submission method registry","timestamp":"2019-01-03T23:36:56.977Z"}
```
Need to see why it was removed

Reply:
Removed here 
https://github.com/CMSgov/qpp-measures-data/pull/269/files#diff-19cc2b2b0a838f96141210f2b77688c9L4712